.travis.yml

```diff
@@ -12,6 +12,7 @@ matrix:
 before_install:
   - export BOTO_CONFIG=/dev/null
 install:
+  - pip install pip==19.0.3
```

Comment:
We're standardizing on 20.0.2 in most places, but this is probably fine for now.

Reply:
somehow the build failed on version >= 20, for python 2.7 and 3.6, I had to specify this.

Reply:
I think there's a narrow window of versions >= 20 that will work.

Reply:
I think I tried 20.0.2 and 20.1.1, went back to the version that was previously working.